packages/react-components/.storybook/preview.js

```diff
@@ -17,7 +17,7 @@ const options = {
       [
         'Introduction',
         'Developer',
-        ['Quick Start', 'Migrating from @fluentui/react v8', 'Styling Components', 'Positioning Components'],
+        ['Quick Start', 'Migrating from @fluentui᜵react v8', 'Styling Components', 'Positioning Components'],
```

Comment:
This is an interesting idea but I do worry a little about reducing searchability (if someone tries to find `@fluentui/` it won't come up with anything), and about someone not realizing why it's a weird slash and changing it back to a normal slash later. But based on the [storybook docs](https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#grouping) it looks usage of a slash to trigger grouping is hardcoded on their side, so I'm not sure what else we could do unless they provide a hook to modify grouping behavior.

Reply:
Yeah, I found the same documentation and also found [this thread](https://github.com/storybookjs/storybook/discussions/12374) where they specify that they don't support escaping characters, so I don't know what else we could do.

Reply:
I can also add a comment indicating why it's a weird slash if that sounds helpful.

Reply:
Huh okay, in that case let's go with the weird slash and add a comment

Reply:
I guess you forgot to add the reasoning into the source as the PR was merged. can you add it please ?

Reply:
in general I'm not a fan of this "hack". 

- should this document be under `Migrations` 
![2022-01-05 at 13 32](https://user-images.githubusercontent.com/1223799/148218303-28ce3392-189b-4ce3-b578-78e96ab9955c.png)

- cant we have the menu item named as `Migrating from @fluentui react v8` until SB implements proper slash escaping ? From my perspective anyone that uses v8 (fluentui/react) knows what v8 means and thus missing exact package name in the menu navigation shouldn't be an issue. WDYT ?

Reply:
Saying `Migrating from @fluentui react v8` could be okay, since "v8" is used alongside the name of the library. But in general we need to be careful about using "v8" as a standalone shorthand in user-facing content since it's also the name of a javascript engine (despite the very different context, it's confused some people in the past).

Reply:
sounds good, thx for additional context !